boards/xtensa/intel_adsp_cavs18/doc/generic_intel_adsp.rst

```diff
@@ -61,6 +61,17 @@ also specify the rimage tool, config and key by:
               --config-dir [path to dir of .toml config file] \
               --key [path to signing key]
 
+
+Note: Sometimes you might have multiple network interfaces. In this case,
+if the cavstool.py cannot start with a correct IP for you, you can check
+your IP address by "ip a" or "ifconfig" command, then specify correct the
+IP address by:
+
+.. code-block:: console
+
+   sudo ./cavstool.py -s 192.168.0.2
```

Comment:
Why no support for `(INADDR_ANY, port)`? This would be so much simpler. Binding can be a nice security feature in complex network configurations but most DUTs will have a single interface which means zero security value.


https://docs.python.org/3/library/socket.html#socket-families

Reply:
Good suggestion, let me take a look, thanks!

Reply:
Hi @marc-hb , I would like to put this in the next update, (include what you suggested using the high level-api, might need some time, thanks!)